core/primitives/src/errors.rs

```diff
@@ -824,16 +825,7 @@ impl Display for InvalidAccessKeyError {
 
 impl std::error::Error for InvalidAccessKeyError {}
 
-#[derive(BorshSerialize, BorshDeserialize, Debug, Clone, PartialEq, Eq, ProtocolSchema)]
-pub struct IntegerOverflowError;
-
-impl std::fmt::Display for IntegerOverflowError {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> Result<(), std::fmt::Error> {
-        f.write_str(&format!("{:?}", self))
-    }
-}
-
-impl std::error::Error for IntegerOverflowError {}
+pub use near_primitives_core::errors::IntegerOverflowError;
```

Comment:
Any reason for this re-export to be here rather than at the top of the file?

Reply:
If any code imported `IntegerOverflowError` from this path instead of the top-level, it will become a breaking change for them, so I'd keep it here

Reply:
ya, but why line 828 of this file, and not lines 1-12? It can still be a re-export. And there are no inline `mod`s in this file that would otherwise modify the path at which this reexport is accessible.